lib/libc/Kconfig

```diff
@@ -56,4 +56,16 @@ config STDOUT_CONSOLE
 	  device, rather than suppressing it entirely. See also EARLY_CONSOLE
 	  option.
 
+# Minimal libc options
+
+config MALLOC_ARENA_SIZE
```

Comment:
if this is going to be minimal libc only, does it make sense to put this in the name as a minimal libc feature?